snyk-monitor/README.md

```diff
@@ -126,6 +126,21 @@ helm upgrade --install ... \
   --set no_proxy=long.domain.name.local,example.local
 ```
 
+## PodSecurityPolicies
+**This should not be used when installing on OpenShift.**
+
+Using PodSecurityPolicies can be achieved by setting the following values in the Helm chart:
+* psp.enabled - default is `false`. set to `true` if PodSecurityPolicy is needed
+* psp.name - default is empty. leave it empty if you want us to install the necessary PodSecurityPolicy. modify it to specify an existing PodSecurityPolicy rather than creating a new one.
```

Comment:
Nit: could we capitalize the start of the sentences, e.g. `Set`, `Leave`, `Modify`

Reply:
DONE